rtc/configuration.md

```diff
@@ -8,361 +8,359 @@ keywords: rtc configuration
+Only the full name is required, but the complete object will be cloned and stored within the RTC caret information. It will then be included with both [`rtc_client_connected`](#rtc_client_connected) and [`rtc_client_disconnected`](#rtc_client_disconnected) callbacks so that the lookup does not need to be repeated.
+
+This provider function will be called once for each connecting client. Clients that reconnect may trigger a new call to the provider function rather than using cached data.
 
 **Type:** `Function`
 
 **Required:** no
 
-### Input fields for `rtc_user_details_provider`
+#### Input fields for `rtc_user_details_provider`
 
 | Field | Type | Description |
 |-------|:----:|-------------|
 | `userId` | `string` | User ID to resolve into user details. |
 
-### Return fields for `rtc_user_details_provider`
+#### Fields to return from `rtc_user_details_provider`
 
 | Field | Type | Description |
 |-------|:----:|-------------|
 | `fullName` | `string` | Full name of user. For example: `"John Doe"`. |
+| any custom field | `any` | Extra user data for use in the [`rtc_client_connected`](#rtc_client_connected) api |
 
-### Example of providing static user details
+#### Example of providing static user details
 
 ```js
 tinymce.init({
-  selector: 'textarea',  // change this value according to your HTML
+  selector: 'textarea',
   plugins: 'rtc',
   rtc_user_details_provider: ({userId}) => Promise.resolve({ fullName: "John Doe" })
 })
 ```
 
-### Example of providing user details from your server
+#### Example of providing user details from your server
 
 ```js
 tinymce.init({
-  selector: 'textarea',  // change this value according to your HTML
+  selector: 'textarea',
   plugins: 'rtc',
   rtc_user_details_provider: ({userId}) => {
-    return fetch('http://yourserver/getUserDetails', {
+    return fetch('/getUserDetails', {
       method: 'POST',
       headers: {
         'Accept': 'application/json',
         'Content-Type': 'application/json'
       },
       body: JSON.stringify({userId})
-    });
+    }).then(response => response.json());
   }
 })
 ```
 
-## `rtc_custom_user_details`
-
-The `rtc_custom_user_details` option allows extra details to be provided about the current user to the other connecting clients. This option should not be used to communicate sensitive information; the authenticity of user data cannot not guaranteed.
+### `rtc_client_info`
 
-This API is designed for use in status flags such as "is the user on a mobile device".
+The `rtc_client_info` option allows status flags from the local editor environment to be provided to other connecting clients via the [`rtc_client_connected`](#rtc_client_connected) API, for example "is the user on a mobile device". This configuration should not be used to communicate sensitive information; the authenticity of the data cannot be guaranteed.
```

Comment:
I don't see any latin. If the word `authenticity` scares someone, they need to talk to a security professional. They probably need to anyway, I just didn't want to say that.

The cost of getting this wrong is users could spoof their identity to other collaborators.

Reply:
This text is matched with the later text which says the same thing from the receiving end:
https://github.com/tinymce/tinymce-docs/pull/1899/files#diff-50e130c2b6f1bd13d6561456b895868f6664f00580c51b65a095375004d7474eR327

I'm happy to adjust but it needs to be the same on both sides.

Reply:
I've adjusted the other side a bit to focus on `accuracy` over `authenticity`.

I'm not 100% sure how to word this now, because really the problem we're describing is that nothing prevents a malicious user from editing their own client info. If sensitive information is available for them to edit, that could mislead other users in the collaboration session.

Reply:
.... no `via` is latin. I had no problem with anything else.

Reply:
Oh. Via is used in a bunch of places (three are still left including this one). I think it's common enough to not call it Latin but I can accept that it doesn't belong in technical docs.

Reply:
one day I'll get'em all.